BitwardenShared/UI/Auth/TwoFactorNotice/Alert+TwoFactorNotice.swift

```diff
@@ -0,0 +1,45 @@
+// MARK: - Alert + TwoFactorNotice
+
+extension Alert {
+    // MARK: Methods
+
+    /// An alert that asks if the user wants to change their email
+    /// in a browser.
+    ///
+    /// - Parameters:
+    ///   - action: The action taken if they select continue.
+    /// - Returns: An alert that asks if the user wants to navigate to the change email page
+    ///
+    static func changeEmailAlert(action: @escaping () -> Void) -> Alert {
+        Alert(
+            title: Localizations.changeAccountEmail,
+            message: Localizations.changeEmailConfirmation,
```

Comment:
🤔 Based on what was discussed internally I think this should be updated to not use the host url but the name of the product so to avoid the logic to see if it's US or EU host.

Reply:
There appears to be a few other places we explicitly name `bitwarden.com` as well, where it might make sense to not use the host URL. I kind of feel like that should be done in a separate PR, though?